Resources/Prototypes/silicon-laws.yml

```
@@ -759,3 +759,11 @@
     Druid: 1
     #Potentially Harmful Lawsets
     Tyrant: 0.25
+    Quarantine: 0.5
+    CCTV: 0.5
+    NTAgressive: 0.5  
+    #Harmful Lawsets
+    Quarantine: 0.5
```

YAMLlint (1.35.1)

[error] duplication of key "Quarantine" in mapping

(key-duplicates)